src/Adliance.Togglr/TogglClient.cs

```diff
@@ -35,53 +36,46 @@ private async Task<IEnumerable<DetailedReportDatum>> DownloadEntries(Configurati
         return detailedReports;
     }
 
-    public async Task DownloadEntriesAndStoreLocally(Configuration configuration)
+    public async Task DownloadEntriesAndStoreLocally()
     {
         var entries = new List<DetailedReportDatum>();
 
-        if (configuration.CacheEntriesUntilYear.HasValue)
+        // check if file with old entries already exists; otherwise download all entries
+        if (!File.Exists("entries.json"))
         {
-            // check if file with old entries already exists; otherwise download the old entries
-            if (!File.Exists("entries_until_" + configuration.CacheEntriesUntilYear + ".json"))
+            for (var i = 2015; i <= DateTime.UtcNow.Year; i++)
```

Comment:
Ich weiß es war vorher schon so, aber auch das Startjahr wäre m. M. n. in einer Konstante besser aufgehoben.